packages/babel-make-styles/src/plugin.ts

```diff
@@ -458,6 +464,13 @@ export const plugin = declare<Partial<BabelPluginOptions>, PluginObj<BabelPlugin
     ...options,
   };
 
+  const ajv = new Ajv();
+  const valid = ajv.validate(configSchema, pluginOptions);
+
+  if (!valid) {
+    throw new Error(`Validation failed for passed config: ${ajv.errorsText(ajv.errors)}`);
+  }
```

Comment:
nit: extract to a separate function since you're instantiating an ajv instance ?

Reply:
Done in a05ed09.